hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+This release fixes several bugs that were introduced to the shrinker in
+:ref:`3.16.24 <v3.66.24>` which would have caused it to behave significantly
```

Comment:
The link text is still mismatched - 16 != 66

Reply:
Argh. So it is.